skip/supplemental_genbank_tests/genbank_upload_quality_test.py

```diff
@@ -215,19 +188,6 @@ def test_for_empty_feature_warnings(self):
         self.assertTrue(empty_warning_count == 0, str(empty_warning_count) + " features had empty warnings.")     
         self.assertTrue(features_with_warnings_count > 0, "No features had warnings.")
 
-#    def test_no_empty_mRNAs(self):
-#        genome = self.__class__.genome
-#        if "mrnas" in genome:
-#            self.assertTrue(len(genome["mrnas"]) > 0, "The mRNA list is empty and is still present.")
-
-#    def test_no_empty_genome_level_warnings(self):
```

Comment:
Why test removed?

Reply:
They are commented out (because they fail). But I'd argue it's helpful to have empty lists because that implies something different than a missing list, namely that the uploader was capable of producing mrnas and warnings but this genome does not have them. For individual features, this can take up a lot of space but I don't see much harm on a genome level.

Reply:
ok fair enough